server/statistics_handler_test.go

```diff
@@ -34,13 +34,18 @@ import (
 )
 
 type testDumpStatsSuite struct {
-	server *Server
-	sh     *StatsHandler
-	store  kv.Storage
-	domain *domain.Domain
+	server     *Server
+	sh         *StatsHandler
+	store      kv.Storage
+	domain     *domain.Domain
+	port       uint
+	statusPort uint
 }
 
-var _ = Suite(new(testDumpStatsSuite))
+var _ = Suite(&testDumpStatsSuite{
+	port:       4012,
+	statusPort: 10099,
```

Comment:
I think special a port here is not a good idea.
We have a mounts of test case in `server/http_handle_test.go`, which use port `4001`.
If we want to make all of them run parallel, specific port for them by `4013` , `4014`, and so on is a wrong way.
Suggestion: use `net.Listen("127.0.0.1:0")` to bind a non-used port. Or assign a rand port( before use it, check if it is used by `net.Dial()`).